### DISCUSSION TITLE
How to modify the data-title with media query on radio button group

### DISCUSSION THREAD

**Discussion by veggie2u** [saadeghi/daisyui#1282] [Q&A] [answered]
I have a button group that shows [ Ft | Pt | Ab ]. Works great. I went with the abbreviations so it fit on mobile. I would now like to use a media query to show [ Full Time | Part Time | Absent ]. After looking at the tailwind docs for [content](https://tailwindcss.com/docs/content) I tried this.

```
data-title="Ft"
class="btn lg:before:content-['Full_Time']"
```

However, I don't see the change in the browser. When I look at the css in chrome dev tools, I see the daisy stuff:

```
.btn-group > input[type="radio"].btn:before {
    content: attr(data-title);
}
```
and mine
```
@media (min-width: 1024px)
<style>
.lg\:before\:content-\[\'Full_Time\'\]::before {
    --tw-content: 'Full Time';
   content: var(--tw-content); -- this line has strikethrough -- overridden
}
```
If I uncheck the daisyui `content: attr(data-title);` line, then mine shows up. It appears that I am not being specific enough to make this happen. Is there some way I can accomplish this with just tailwind? Would I perhaps need to not use daisy here and just paste in the tailwind stuff daisy uses so that I can make sure I can override it? Or if there is a better suggestion I am all ears. I had also considered JS, but I thought it better to leave the screen size media queries to tailwind.

**Comment by saadeghi** [comment, marked as answer]
That's challenging... but possible. This ugly class name will help you:
```
[.btn-group>input[type='radio']&]:lg:before:content-['Full_Time']
```
https://play.tailwindcss.com/LdHV9rY7VS

I would use [`btn-group-vertical`](https://daisyui.com/components/button-group/) for mobile tho. Those abbreviations (`Ft`, `Pt` etc.) are not user friendly.

**Reply by veggie2u** [reply]
This worked great. I wanted full names on desktop because the abbreviations are less clear. I didn't think full circle, half circle, empty circle would work either. Unfortunately vertical space isn't there. But it is an admin app, and after a couple of days everyone will know.
